content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,283 @@
+`istio-system` namespace:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: ingress
+ namespace: istio-system
+spec:
+ selector:
+   matchLabels:
+     app: istio-ingressgateway
+ rules:
+ # omitted
+{{< /text >}}
+
+### Comparision
+
+The following table highlights the key differences between the old `v1alpha1` RBAC policies and
+the new `v1beta1` authorization policy.
+
+| Feature | `v1alpha1` RBAC policy | `v1beta1` Authorization Policy |
+|---------|------------------------|--------------------------------|
+| Number of CRDs | Three: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding` | Only One: `AuthorizationPolicy` |
+| Mechanism to specify where to apply the policy | Specify by **service** | Specify by **workload** |
+| Deny-by-default behavior | Enabled **explicitly** by configuring `ClusterRbacConfig` | Enabled **implicitly** with `AuthorizationPolicy` |
+| API stability | `alpha`: **No** backward compatible | `beta`: backward compatible **guaranteed** |
+| `"*"` in policy | Match all contents (empty and non-empty) | Match non-empty contents only |
+| Ingress/Egress gateway support | Not supported | Supported |
+
+Beyond all the differences, the `v1beta1` policy is enforced by the same engine in Envoy
+and supports the same authenticated identity (mutual TLS or JWT), condition and other
+primitives (e.g. IP, port and etc.) as the `v1alpha1` policy.
+
+See [authorization concept page](/docs/concepts/security/#authorization) for a detailed
+in-depth explanation of the `AuthorizationPolicy`.
+
+## Migration
+
+Istio 1.4 supports both `v1alpha` and `v1beta1` policies at the same time. The `v1beta1` policy
+overrides the `v1alpha1` policy if they are both applied to the same workload:
+
+1. If any `v1beta1` policies apply to the workload, the `v1beta1` policy is used and
+   the `v1alpha1` policies on the same workload will all be ignored
+1. If no `v1beta1` policy applies to the workload, the `v1alpha1` policy is used if
+   there is any
+
+{{< warning >}}
+This means when migrating to the `v1beta1`, you need to make sure the converted
+`v1beta1` policy covers all the `v1alpha1` policies applied to the same workload. The
+`v1alpha1` policies applied for the workload will be ignored after you applied the `v1beta1` policies.
+{{< /warning >}}
+
+It's recommended to migrate the `v1alpha1` policy workload-by-workload to make sure you don't
+forget any `v1alpha1` policies:
+
+1. For a given workload, find out all the `v1alpha1` policies applied to it
+1. Convert all the `v1alpha1` policies to `v1beta1` for the workload
+1. Apply the `v1beta1` policy and monitor the traffic to make sure the new policy is working as expected
+1. Continue the process for the next workload until all `v1alpha1` policies are converted
+1. Now you can remove all your `v1alpha1` policies
+
+To help ease the migration, Istio provides the command `istioctl x auth migrate` to automate the
```

Comment:
link to istioctl reference page for the command? maybe

Reply:
Yes, we should, the page is not updated yet (the command is still marked hide). will update when it's done.